### PR TITLE
docs(setup): add type checker commands on setup guide

### PR DESCRIPTION
**Pull request by TheLarkInn**
**What kind of change does this PR introduce?**
Docs/Readme's

**Did you add tests for your changes?**
N/A

**If relevant, link to documentation update:**
N/A

**Summary**
 This adds how to run the typechecker in the setup.md guide. 

**Does this PR introduce a breaking change?**
Nope, just happy typecheckers

**Other information**

I love lamp.

### CODE REVIEW

File: _SETUP.md [hunk 1]
```diff
@@ -21,8 +21,20 @@ yarn link
...
 ```
 
-To run the entire test suite use:
+### To run the entire test suite use:
 
 ```bash
 yarn test
 ```
+
+### To run the typechecker use:
```
Comment (sokra) [inline _SETUP.md]: To run only the typechecker

File: _SETUP.md [hunk 2]
```diff
@@ -21,8 +21,20 @@ yarn link
...
+
+```bash
+yarn type-lint
+```
+
+or incremental as you add JSDoc Annotations or make changes
+
+```bash
+yarn type-lint --watch
+```
```
Comment (sokra) [inline _SETUP.md]: Also add notes about the other commands:

`yarn lint` `yarn code-lint` `yarn test:integration` `yarn test:integration --watch` `yarn test:unit` `yarn pretty`

Comment (TheLarkInn) [reply _SETUP.md]: Hehe, I saw this coming. :joy:

**Comment by sokra** [comment]
Thanks

**Pull request merged**